fluss-common/src/main/java/org/apache/fluss/row/compacted/CompactedRowReader.java

```diff
@@ -325,7 +326,9 @@ static FieldReader createFieldReader(DataType fieldType) {
                 DataType elementType = ((ArrayType) fieldType).getElementType();
                 fieldReader = (reader, pos) -> reader.readArray(elementType);
                 break;
-
+            case MAP:
+                fieldReader = (reader, pos) -> reader.readMap();
```

Comment:
should pass `keyType` and `valueType` into it, otherwise, we can't support `Map<K, Row>` nested type.